libsolutil/Keccak256.cpp

```diff
@@ -69,25 +69,25 @@ static uint64_t const RC[24] = \
 #define REPEAT6(e) e e e e e e
 #define REPEAT24(e) REPEAT6(e e e e)
 #define REPEAT5(e) e e e e e
-#define FOR5(v, s, e) \
+#define FOR5(type, v, s, e) \
 	v = 0;            \
-	REPEAT5(e; v += s;)
+	REPEAT5(e; v = static_cast<type>(v + s);)
```

Comment:
I think you might get away without adding `type` if you added `u` suffix to the constants that get passed into the `s` parameter.